src/components/RevokeContainer/RevokeDocumentDropZone/RevokeDocumentDropZone.tsx

```diff
@@ -3,7 +3,6 @@ import { FunctionComponent } from "react";
 import { useDropzone } from "react-dropzone";
 import { readFileAsJson } from "../../../common/utils";
 import { getLogger } from "../../../utils/logger";
-import { Title } from "../../UI/Title";
 import { Wrapper } from "../../UI/Wrapper";
```

Comment:
if title no longer used, can delete that entire folder

Reply:
will remove in next PR, still used some places